build/Targets/Imports.targets

```diff
@@ -90,6 +90,11 @@
   </Choose>
 
   <Choose>
+    <When Condition="'$(TargetFrameworkIdentifier)' == '.NETPortable' AND '$(OS)' != 'Windows_NT'">
+      <PropertyGroup>
+        <MSBuildTargetsFilePath>$(MSBuildExtensionsPath32)\Extensions\Microsoft\Portable\$(TargetFrameworkVersion)\Microsoft.Portable.$(MSBuildTargetsLanguageName).targets</MSBuildTargetsFilePath>
+      </PropertyGroup>
+    </When>
```

Comment:
https://github.com/Microsoft/msbuild/issues/1533

Next change will get a link to that added into the targets file.